source/Weapon.h

```diff
@@ -50,6 +50,11 @@ class Weapon {
 		Point offset;
 	};
 
+	enum class InaccuracyModes{
+		Triangular,
+		Uniform,
+		Normal
+	};
```

Comment:
Put these in their own file. The enum name should be something like `Distributions`

Reply:
Seems like that would just be a header file?  Can't see a need for a .cpp file unless the logic gets moved out of Random as you suggested *and* doesn't get moved into Projectile::Inaccuracy().

Reply:
Addressed in 2f317f286845965478e3ef55953956d053b90d21, with just a header file as mentioned above.  I think I did something wrong with the file addition as the checks don't seem to be passing.  It did build and run but obviously something more specific is incorrect.

Reply:
quyykk helped me find i what file was deficient and I fixed the failing check.